boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
I’ve never loved the “env” way of shebanging. Does the Jenkins server have /usr/bin/python3? I sure would love to remain explicit if that’s possible.

Reply:
You can also change how the script is started to `python3 gen...` instead. That is more likely to work than the `env` approach.

Reply:
Even better, let's run the script in a py3 container.

(We've just learned py3 doesn't even exist on the jenkins server :mindblown:)